src/Shared/ServerInfrastructure/StringUtilities.cs

```diff
@@ -70,27 +67,59 @@ private static unsafe void GetAsciiStringNonNullCharacters(Span<char> buffer, In
             }
         }
 
+        private static readonly SpanAction<char, IntPtr> s_getAsciiStringNonNullCharacters = GetAsciiStringNonNullCharacters;
```

Comment:
Nit: Can we move the static `SpanAction` fields to the top of the class?

Reply:
Done with efe80e78c9925160f5aa602ada1c6b4d9fe9fb73

---

A note for the rationale behind the placement:
As these delegates are used in only one special place (to avoid the compiler generating the cachable display-class and the null-check) these were placed near to the usage to have "compact code".